grin/app/GrinCLI.hs

```diff
@@ -118,6 +123,34 @@ pipelineOpts =
   <|> flg ConfluenceTest "confluence-test" "Checks transformation confluence by generating random two pipelines which reaches the fix points."
   <|> flg PrintErrors "print-errors" "Prints the error log"
 
+maybeRenderingOpt :: String -> Maybe RenderingOption
+maybeRenderingOpt = M.parseMaybe renderingOpt
+
+renderingOpt :: M.Parsec Void String RenderingOption
+renderingOpt = Simple        <$ M.string "simple"
+           <|> WithExternals <$ M.string "with-externals"
+
+{- NOTE: Cannot use default in some/many combinators.
+   The library considers default value as:
+     "nothing is given, then use default",
+   but we want it to behave like:
+    "if the flag is parsed, and no argument is given, then use default"
+-}
+printGrinWithOpt :: Parser PipelineStep
+printGrinWithOpt = flip PrintGrin id <$> option (maybeReader maybeRenderingOpt)
+  ( long "print-grin"
+  <> help "Print the actual grin code with a given rendering option [simple | with-externals]"
+  -- <> showDefaultWith (camelToDashed . show)
```

Comment:
Please remove the commented out code parts and the non-used helper function.

Reply:
Done.